docs/source/guide/install_enterprise_k8s.md

```diff
@@ -0,0 +1,418 @@
+
+Your Kubernetes cluster can be self-hosted or installed somewhere such as Amazon EKS. See the Amazon tutorial on how to [Deploy a Kubernetes Application with Amazon Elastic Container Service for Kubernetes](https://aws.amazon.com/getting-started/hands-on/deploy-kubernetes-app-amazon-eks/) for more about deploying an app on Amazon EKS.
+
+## Prepare the Kubernetes cluster
+
+Before installing Label Studio Enterprise, prepare the Kubernetes cluster with [kubectl](https://kubernetes.io/docs/reference/kubectl/). 
+
+1. Create a key to pull the latest Label Studio Enterprise image from the Docker registry. From the command line, run the following:
+    ```shell
+    kubectl create secret docker-registry heartex-pull-key \
+        --docker-server=https://index.docker.io/v2/ \
+        --docker-username=heartexlabs \
+        --docker-password=<CUSTOMER_PASSWORD>
+    ```
+2. Create the Label Studio Enterprise license as a Kubernetes secret. You can specify it as a file or as a specific URL. 
+   From the command line, specify the license as a file:
+   ```shell
+   kubectl create secret generic lse-license --from-file=license=path/to/lic
+   ```
+   Or from the command line, specify the license as a URL:
+   ```shell
+   kubectl create secret generic lse-license --from-literal=license=https://lic.heartex.ai/db/<CUSTOMER_LICENSE_ID>
+   ```
+
+## Configure the Helm chart for Label Studio Enterprise
+
+Install Label Studio Enterprise and set up a PostgreSQL or Redis database to store relevant Label Studio Enterprise configurations and annotations using the Helm chart. You must configure specific values for your deployment in a YAML file that you specify when installing using Helm.
+
+
+A minimal installation of LSE requires the following values:
+
+```yaml
+global:
+  imagePullSecrets:
+    # Defined with earlier kubectl command
+    - name: heartex-pull-key
+  
+  # [Enterprise Only] This value refers to a Kubernetes secret that you 
+  # created that contains your enterprise license.
+  enterpriseLicense:
+    secretName: "lse-license"
+    secretKey: "license"
+  pgConfig:
+    # PostgreSql instance hostname
+    host: "postgresql"
+    # PostgreSql database name
+    dbName: "my-database"
+    # PostgreSql username
+    userName: "postgres"
+    # PostgreSql password secret coordinates within Kubernetes secrets 
+    password:
+      secretName: "postgresql"
+      secretKey: "postgresql-password"
+
+  redisConfig:
+    # Redis connection string
+    host: redis://host:port/db
+  
+# Ingress config for Label Studio
+app:
+  ingress:
+    host: studio.yourdomain.com
+# if you have a tls cert, uncomment the next section
+#    tls:
+#      - secretName: ssl-cert
+#        hosts:
+#          - studio.yourdomain.com
+```
+
+Adjust the included defaults to reflect your environment and copy these into a new file and save it as `lse-values.yaml`. 
+
+
+<br/>
+{% details <b>Click to expand the full list of configurable options</b> %}
```

Comment:
I hid the full values.yml file from the repo behind this expander, since it's part of the LSE repo and isn't public.

Reply:
that's ok